app/src/main/kotlin/com/wire/android/ui/home/newconversation/contacts/ContactsViewModel.kt

```diff
@@ -0,0 +1,63 @@
+package com.wire.android.ui.home.newconversation.contacts
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.wire.android.navigation.NavigationManager
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@HiltViewModel
+class ContactsViewModel @Inject constructor(private val navigationManager: NavigationManager) : ViewModel() {
+
+    val contactsState by mutableStateOf(
+        ContactsState(
+            contacts = listOf(
+                Contact(
+                    "This is first contact"
+                ), Contact(
+                    "test2"
+                ), Contact(
+                    "test3"
+                ),
+                Contact(
+                    "test1"
+                ), Contact(
+                    "test2"
+                ), Contact(
+                    "test3"
+                ),
+                Contact(
+                    "test1"
+                ), Contact(
+                    "test2"
+                ), Contact(
+                    "test3"
+                ),
+                Contact(
+                    "test1"
+                ), Contact(
+                    "test2"
+                ), Contact(
+                    "test3"
+                ),
+                Contact(
+                    "test1"
+                ), Contact(
+                    "test2"
+                ), Contact(
+                    "This is last contact"
+                )
+            )
```

Comment:
> we could do it like this
> 
> ```
> buildList<Contact> {
>         Contact("This is first contact")
>         (1..13).map {  Contact("test $it") }
>         Contact("This is last contact")
> }
> ```

https://github.com/wireapp/wire-android-reloaded/pull/410/files#r820912381

I think the correct calls are `add(item)` and `addAll(items)`:

```kotlin
buildList<Contact> {
        add(Contact("This is first contact"))
        addAll((1..13).map {  Contact("test $it") })
        add(Contact("This is last contact"))
}
```